packages/@react-types/overlays/src/index.d.ts

```diff
@@ -23,12 +23,36 @@ export type SizeAxis = 'width' | 'height';
 export type PlacementAxis = Axis | 'center';
 
 export interface PositionProps {
+  /** The placement of the element with respect to its anchor element. */
   placement?: Placement,
+  /** 
+   * The placement padding that should be applied between the element and its 
+   * surrounding container. 
+   */
   containerPadding?: number,
+  /** 
+   * The additional offset that should be applied between the element and its 
+   * anchor element. Offset is applied in the same direction in which the element 
```

Comment:
I wonder if we should use the language that the specs uses https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox
Offset is applied along the main axis, crossoffset is applied along the cross axis
it's a little bit like using the word in the definition... but it should be an understood term